FrontDesk/tests/UnitTests/Core/AggregatesEntities/AppointmentTests/Appointment_UpdateTime.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using AutoFixture;
+using FrontDesk.Core.Aggregates;
+using PluralsightDdd.SharedKernel;
+using Xunit;
+
+namespace UnitTests.Core.AggregatesEntities.AppointmentTests
+{
+  public class Appointment_UpdateTime
+  {
+    private Fixture _fixture = new Fixture();
+
+    [Fact]
+    public void SuccessUpdateTime()
```

Comment:
Test names should read like a sentence when combined with their class name:
Example:
`Appointment_UpdateRoom.UpdatesRoomIdAndAddsEventGivenNewRoomId`

I don't know what this sentence means:
`Appointment_UpdateTime.SuccessUpdateTime`